pkg/agent/openflow/multicast.go

```diff
@@ -75,13 +77,44 @@ func (f *featureMulticast) replayFlows() []binding.Flow {
 	return getCachedFlows(f.cachedFlows)
 }
 
+// multicastIngressPodMetricFlows generates the flows to forward multicast ingress packets before
```

Comment:
This flow is to help collect ingress metrics but not forward packets. Please refine the comments

Reply:
updated